src/test/java/com/cloudbees/hudson/plugins/folder/FolderTest.java

```diff
@@ -424,20 +424,20 @@ private void copyViaHttp(Folder f, JenkinsRule.WebClient wc, String fromName, St
     }
 
     @Issue("JENKINS-58282")
-    @Test public void shouldHaveHealthMetricConfiguredGloballyOnCreation() throws Exception {
-        assertThat("by default, global configuration should have all folder health metrics",
-                AbstractFolderConfiguration.get().getHealthMetrics(), hasSize((int) FolderHealthMetricDescriptor.all().stream().filter(d -> d.createDefault() != null).count()));
+    @Test public void shouldNotHaveHealthMetricConfiguredGloballyOnCreation() throws Exception {
```

Comment:
Is it possible to create a test that validates the metric appears when the property is set? (basically the previous test plus setting the property to true).

Reply:
@fbelzunc I think @varyvol has made a great point!

Reply:
@varyvol @fcojfernandez Sure, good idea! Do you want me to set the property with `System.setProperty` or there is something better I am not aware of?

Reply:
`System.setProperty` is OK. Just remember to clear the value after the test has finished.